spec/requests/citizens/bank_transactions_spec.rb

```diff
@@ -32,17 +32,23 @@
       expect(response).to redirect_to(citizens_identify_types_of_income_path)
     end
 
-    context 'with JSON request' do
-      let(:headers) { { 'ACCEPT' => 'application/json' } }
+    context 'with ajax request' do
+      subject do
+        patch(
+          remove_transaction_type_citizens_bank_transaction_path(bank_transaction),
+          headers: headers,
+          xhr: true
+        )
+      end
```

Comment:
Another to way to implement this:
![image](https://user-images.githubusercontent.com/482806/54742748-28155e00-4bba-11e9-9772-a963e574882a.png)

I think it's a bit nicer as it's more obvious that the only difference is that the request is `XHR`